UI/js/new__blog.js

```diff
@@ -17,10 +17,17 @@ new_blog_form.addEventListener("submit", (e) => {
     } else {
         let notification = document.querySelector("#error__message");
         notification.classList.add("alert__message");
-        notification.textContent = "Your message has been sent thank you.";
-        setTimeout(() => {
+        notification.textContent = "Your article has been posted thank you.";
```

Comment:
An average user don't understand what an article being posted mean. Post might mean create for you but normal users would appreciate you saying **`created`** instead of **`posted`**.

Reply:
I am going to change Created instead of post